e2e/src/authentication/login.e2e-spec.ts

```diff
@@ -0,0 +1,80 @@
+import { LoginPage } from './login.po';
+import { browser, logging, until } from 'protractor';
+
+describe('Login', () => {
+  let page: LoginPage;
+
+  beforeEach(() => {
+    page = new LoginPage();
+  });
+
+  describe('Given valid credential', () => {
```

Comment:
Minor typo:

```suggestion
  describe('Given valid credentials', () => {
```

Also, recheck all below as you often `the` e.g. `'enables the form submit button' and so on